admin/class-wizard-notification.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ * @since   14.1
```

Comment:
I think we decided to use `@since` notations only for things like API methods, not in other cases.